plugins/examples/tables/README.md

```diff
@@ -0,0 +1,14 @@
+# Tables
+
+This example plugin overrides the pod list view's table to add a context menu.
```

Comment:
There's a place in docs/plugins/development/functionality.md to add more detail. Including a screenshot in docs/plugins/development/images/

Reply:
Adding that.